packages/synapse-react-client/src/components/DataGrid/SynapseGrid.tsx

```diff
@@ -238,7 +244,7 @@ const SynapseGrid = forwardRef<
         // and adding rows to the model via the api
         for (let i = operation.fromRowIndex; i < operation.toRowIndex; i++) {
           const rowArr = model.api.arr(['rows'])
-          rowArr.ins(i, [{ data: s.vec(s.con('')), metadata: {} }])
+          rowArr.ins(i, [{ data: s.vec(s.con('')), metadata: s.obj({}) }])
```

Comment:
As John mentioned, we shouldn't need to supply this in the client.

```suggestion
          rowArr.ins(i, [{ data: s.vec(s.con('')) }])
```